src/restful/controllers/applicationsController.js

```diff
@@ -14,7 +14,7 @@ async function createRoom(db, roomId, user_id, user_name, user_image_url, creato
     id: roomId,
     createdAt: new Date().toString(),
     client: user_id,
-    brand: creator_id,
+    client: creator_id,
```

Comment:
We now have 2 fields for client, one should be creator and one should be client. 

Just for clarification the creator user type can apply for opportunities, whereas clients are the users who post opportunities and create contracts.

Reply:
Yep just pushed up the change